applications: nrf_desktop: Documentation improvements (#9481)
nrfconnect/sdk-nrf: applications/nrf_desktop/README.rst

   Some of the nRF Desktop configurations also include `Fast Pair`_ payload in the Bluetooth LE advertising data to simplify pairing the nRF Desktop peripherals with Android hosts.
   These configurations apply further modifications that are needed to improve the user experience.

>    See :ref:`nrf_desktop_bluetooth_guide_fast_pair` documentation section for details.

Reviewer: ```suggestion
   See the :ref:`nrf_desktop_bluetooth_guide_fast_pair` documentation section for details.
```

This API is used only by the application modules that handle such connections.
The information about peer and connection state is propagated to other application modules using :ref:`app_event_manager` events.

> The :ref:`CONFIG_DESKTOP_BT <config_desktop_app_options>` Kconfig option enables Bluetooh connectivity support in the nRF Desktop.

Reviewer: ```suggestion
The :ref:`CONFIG_DESKTOP_BT <config_desktop_app_options>` Kconfig option enables support for Bluetooth connectivity in the nRF Desktop.
```

> The :ref:`CONFIG_DESKTOP_BT <config_desktop_app_options>` Kconfig option enables Bluetooh connectivity support in the nRF Desktop.

Reviewer 2: This is already mentioned in the Bluetooth configuration section

Author: I think that it may still be worth to mention it here as user may not always read whole documentation

Author: Apart from that, it may work as nice one sentence of general introduction

* :kconfig:option:`CONFIG_BT_MAX_PAIRED`

  * nRF Desktop central: The maximum number of paired devices is greater than or equal to the maximum number of simultaneously connected peers.

>     The :kconfig:option:`CONFIG_BT_MAX_PAIRED` is by default set to :ref:`CONFIG_DESKTOP_HID_DONGLE_BOND_COUNT <config_desktop_app_options>`

Reviewer: ```suggestion
    The :kconfig:option:`CONFIG_BT_MAX_PAIRED` is by default set to :ref:`CONFIG_DESKTOP_HID_DONGLE_BOND_COUNT <config_desktop_app_options>`.
```

>     The :kconfig:option:`CONFIG_BT_MAX_PAIRED` is by default set to :ref:`CONFIG_DESKTOP_HID_DONGLE_BOND_COUNT <config_desktop_app_options>`

Reviewer: ```suggestion
    The :kconfig:option:`CONFIG_BT_MAX_PAIRED` is by default set to :ref:`CONFIG_DESKTOP_HID_DONGLE_BOND_COUNT <config_desktop_app_options>`.
```

Author: Duplicate comment, marked as resolved.

* :kconfig:option:`CONFIG_BT_MAX_CONN`

  * nRF Desktop central: This option is set to the maximum number of simultaneously connected devices.

>     The :kconfig:option:`CONFIG_BT_MAX_CONN` is by default set to :ref:`CONFIG_DESKTOP_HID_DONGLE_CONN_COUNT <config_desktop_app_options>`

Reviewer: ```suggestion
    The :kconfig:option:`CONFIG_BT_MAX_CONN` is by default set to :ref:`CONFIG_DESKTOP_HID_DONGLE_CONN_COUNT <config_desktop_app_options>`.
```
